tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/pom.xml

```diff
@@ -8,6 +8,14 @@
 		<version>1.1.0.BUILD-SNAPSHOT</version>
 	</parent>
 
+	<!--Can remove these specific version when the main project updates netty
+	and reactor dependencies-->
+	<properties>
+		<reactor.version>3.0.2.RELEASE</reactor.version>
```

Comment:
If you moved that here, doesn't that mean that something should have changed in the main project?